docs/en/stack/ml/limitations.asciidoc

```diff
@@ -201,10 +201,11 @@ process. For more information, see <<stopping-ml>> and
 [float]
 === Rollup indices and index patterns are not supported
 
-Rollup indices and index patterns cannot be used in machine learning jobs or 
-{dfeeds}. This limitation applies irrespective of whether you create the jobs in 
-{kib} or by using APIs. In {kib}, if you select an index, saved search, or index 
-pattern that uses the Rollup feature, the {ml} job creation wizards fail. 
+Rollup indices and index patterns cannot be used in {ml} {dfeeds}. In {kib}, the
+{ml} job creation wizards will not allow you to select an index pattern that uses
+the rollup feature. If you select a saved search that uses a rollup index or add
+a rollup index directly in the {dfeed} configuration JSON then the {ml} job
+creation wizards will fail.
```

Comment:
This description leaves me wondering whether I can use rollup indices and index patterns in data frame analytics jobs if I create them via the API (since the limitation now only mentions data feeds and Kibana scenarios).

Reply:
Since this PR was first created the limitations file I changed has moved into a `ml/anomaly-detection` sub-directory.  Does that solve the immediate problem, because it's now clear the limitations it lists apply to anomaly detection?

I think there's actually a broader problem, which is that we now have 3 categories of ML limitations:

1. Limitations that apply to everything the ML plugin does, e.g. requiring SSE 4.2 instructions
2. Limitations that apply only to anomaly detection, e.g. anything that refers to datafeeds
3. Limitations that apply only to data frame analytics, e.g. not having any UI - I don't think we've started the effort to collect and document this 3rd category of limitations